openstack/keystone/templates/seed.yaml

```diff
@@ -1,8 +1,7 @@
-{{- if .Release.IsUpgrade }}
```

Comment:
In an 'ordinary' keystone installation the seeds won't work on install - but it's okay I guess, the seeder will re-try to apply the seeds.

Reply:
yes, correct. The problem is that now we have to run upgrade / 2nd deployment before the seeds even appear. Looking at other services, almost nobody seem to have this clause right now.